src/modules/bulk-minter/components/RoyaltiesCreators.tsx

```diff
@@ -227,9 +234,19 @@ const CreatorsRow = ({
       >
         {isHolaplex
           ? 'Hola Community Fund'
+          : charityProps?.displayName 
+          ? charityProps.displayName
           : creatorAddress.slice(0, 4) + '...' + creatorAddress.slice(creatorAddress.length - 4)}
       </Paragraph>
-      {!isHolaplex && (
+      {charityProps?.isCharity && (
+        <FeatherIcon
+          className="creator-row-icon"
+          icon="external-link"
+          onClick={() => {window.open(CHANGE_BASE_URL+creatorAddress, "_blank")}}
+        />
```

Comment:
I think it would be better to wrap this in a link rather than bind a specialized onClick that imitates a link. In other words, use a link for thinks that are link-like.

```suggestion
    <a href={`${CHANGE_BASE_URL+creatorAddress}`} target="_blank">
        <FeatherIcon
          className="creator-row-icon"
          icon="external-link"
        />
  </a>
```